x-pack/plugins/security_solution/public/management/cypress/e2e/mocked_data/automated_response_actions/no_license.cy.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { generateRandomStringName } from '@kbn/osquery-plugin/cypress/tasks/integrations';
+import { APP_ALERTS_PATH } from '../../../../../../common/constants';
+import { closeAllToasts } from '../../../tasks/toasts';
+import { fillUpNewRule } from '../../../tasks/response_actions';
+import { login, loginWithRole, ROLE } from '../../../tasks/login';
+import type { ReturnTypeFromChainable } from '../../../types';
+import { indexEndpointHosts } from '../../../tasks/index_endpoint_hosts';
+import { indexEndpointRuleAlerts } from '../../../tasks/index_endpoint_rule_alerts';
+
+describe('No License', { env: { ftrConfig: { license: 'basic' } } }, () => {
```

Comment:
what not aware of this functionality - where a `describe()` can take in 3 arguments, where the second argument is some sort of configuration for the runtime env (do I have that right?)

Is this new and/or only available to our test suites?  (any chance you can add a topic to our weekly eng. hour to just introduce us to it?)

Reply:
Oh yeah, this functionality is pretty cool, and also quite new because @szwarckonrad and @patrykkopycinski introduced this while doing the parallel cypress tests. 

I don't want to gather everyone just to talk about this, so leaving a short information below just to describe the thing. However if @szwarckonrad or @patrykkopycinski wants to give a broader introduction to new cypress approach during the office hours that could be very informative 🤞 

https://github.com/elastic/kibana/blob/2b496fa493c7d66cf308f147d3a59398c4aad865/x-pack/plugins/security_solution/scripts/run_cypress/parallel.ts#L149 It parses file config (on CI it does that automatically, but locally, developer has to specify `--spec` file and it works :) if no spec file is provided it would take the first file in cypress suite).

Then in https://github.com/elastic/kibana/blob/2b496fa493c7d66cf308f147d3a59398c4aad865/x-pack/plugins/security_solution/scripts/run_cypress/parallel.ts#L284 I added setting up license based on config from test file. Recently some folks has started using this to `enableExperimental` which is alsoooo so cool :)